CeeFM/Decideify
# Building out foundation for editing and deleting suggestions from your saved database

**Description**
New buttons are added to each suggestion where they are listed in Carousels:
- Two buttons for toggling whether the user recommends a specific suggestion or not
- One X button added to the top of each suggestion that will serve as a delete button and remove the suggestion from their saved suggestions

**Type of Change**


**Testing Instructions**

- Clone this project’s repository to your machine
  -If you’ve already done this and are currently working on a different branch of the project, be sure to `git add` and `git commit` whatever you’re working on on your branch, before you attempt to switch branches
  -Then `git checkout main` to switch to the project’s main branch)
- Navigate to the root directory for this repo (yourParentFolderName/Decideify)
- `git fetch -a` to retrieve the current remote-tracked branches for the project, you should see a branch named `cm-suggestions` as one of the available branches
- `git checkout cm-suggestions` to switch to my branch
- If you don't see the most recent version of my files, `git pull origin cm-suggestions`
- `Start Decideify.sln` to open the project in Visual Studio
- If you haven’t created the Decideify database, please do so in Visual Studio and use the following SQL scripts to populate it accordingly:
  - Start a new query in your database and copy in this SQL script before running to fill out tables and columns: [First SQL Script for Creating DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Create_DB.sql)
  - ctrl + a and delete what is in that query and replace it with this SQL script before running to seed the database: [Second SQL Script for Seeding DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Seed_Users_DB.sql)

- Click the execute with debugger solid green button to launch the app
- Log in as coolin@gmail.com with password thistbh
- On every content hub page and on the mylibrary.js component / page, you should see buttons that say YES and NO below each suggestion under a header that says "Recommend?". You should also see a red X button in the top left corner of each suggestion.
- None of these buttons actually work how they are intended to work yet, but that is next! For now we just wanted to add them.
- Also, the carousels now have dedicated PREVIOUS and NEXT buttons below each Carousel that will handle the pagination through each carousel of suggestions. This was implemented because the native previous and next buttons on the bootstrap/ reactstrap carousel was not allowing interaction with the aforementioned recommendation and delete buttons on certain suggestions displayed in the carousel.
- The Previous and Next buttons DO work, and should cycle you through the different saved suggestions as they did previously.

Checklist
